src/window.rs

```diff
@@ -69,12 +69,15 @@ impl Drop for Window {
 pub struct WindowId(pub(crate) platform_impl::WindowId);
 
 impl WindowId {
-    /// Returns a dummy `WindowId`, useful for unit testing. The only guarantee made about the return
-    /// value of this function is that it will always be equal to itself and to future values returned
-    /// by this function.  No other guarantees are made. This may be equal to a real `WindowId`.
+    /// Returns a dummy `WindowId`, useful for unit testing.
+    ///
+    /// # Safety
+    /// The only guarantee made about the return value of this function is that
+    /// it will always be equal to itself and to future values returned by this function.
+    /// No other guarantees are made. This may be equal to a real `WindowId`.
```

Comment:
```suggestion
    /// # Safety
    ///
    /// The only guarantee made about the return value of this function is that
    /// it will always be equal to itself and to future values returned by this function.
    /// No other guarantees are made. This may be equal to a real `WindowId`.
```
nit: you used a space between the heading and the body in another doc comment

Reply:
fixed